trainer/diffusers_trainer.py

```diff
@@ -534,8 +540,12 @@ def main():
         print("RESOLUTION:", args.resolution)
 
     if args.hf_token is None:
-        args.hf_token = os.environ['HF_API_TOKEN']
-        print('It is recommended to set the HF_API_TOKEN environment variable instead of passing it as a command line argument since WandB will automatically log it.')
+        try:
+            args.hf_token = os.environ['HF_API_TOKEN']
+            print('It is recommended to set the HF_API_TOKEN environment variable instead of passing it as a command line argument since WandB will automatically log it.')
```

Comment:
I know that the warning here was in the original code but it's redundant, how about printing the warning if `hf_token` is passed as an argument?

Reply:
I would even go further and abort if it was passed like this in combination with wandb.
Alternatively manual user input like "are you really sure you want to show everyone your hf token?".
The warning will come too late since the token will have been logged already if everything else was in order.

Reply:
> I would even go further and abort if it was passed like this in combination with wandb. Alternatively manual user input like "are you really sure you want to show everyone your hf token?". The warning will come too late since the token will have been logged already if everything else was in order.

is it okay if I add an auto confirm flag too?
I personally just want to run one command rather than having to input every time